master/internal/experiment.go

```diff
@@ -804,7 +806,7 @@ func (e *experiment) setPriority(ctx *actor.Context, priority *int, forward bool
 			Handler:  ctx.Self(),
 		}).(type) {
 		case nil:
-		case rm.ErrUnsupported:
+		case rmerrors.ErrUnsupported:
```

Comment:
open to better naming for any of the packages, didn't put an excessive amount of thought into it.